React/Views/RCTActivityIndicatorView.m

```diff
@@ -66,7 +66,7 @@ - (void)setActivityIndicatorViewStyle:(UIActivityIndicatorViewStyle)activityIndi
   }
 }
 
-- (void)setColor: (UIColor*)color
+- (void)setColor: (RCTUIColor*)color
```

Comment:
Missing the TODO comment

Reply:
This change is already inside a ` // [TODO(macOS ISS#2323203)` ... ` // ]TODO(macOS ISS#2323203)` block.